pkg/scaffold/v2/metricsauth/auth_proxy_patch.go

```diff
@@ -40,8 +40,8 @@ func (f *AuthProxyPatch) GetInput() (input.Input, error) {
 	return f.Input, nil
 }
 
-const kustomizeAuthProxyPatchTemplate = `# This patch inject a sidecar container which is a HTTP proxy for the controller manager,
-# it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+const kustomizeAuthProxyPatchTemplate = `# This patch inject a sidecar container which is a HTTP proxy for the 
```

Comment:
I'm quite confused why some template need `// nolint:lll` while others don't.

Reply:
I just broke the first line put the text bellow.

Reply:
It is not crossing the size any more.